packages/hooks/src/utils/number.ts

```
@@ -63,3 +63,20 @@ export function sub(num1: number, num2: number) {
   const maxPrediction = Math.max(getNumberPrecision(num1), getNumberPrecision(num2));
   return Number(number.toFixed(maxPrediction));
 }
+
+export function toNum(v: number | string | undefined): number {
+  if (v === undefined || v === null) {
+    return 0;
+  }
+
+  if (typeof v === 'number') {
+    return isNaN(v) ? 0 : v;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)